### ISSUE TITLE
Web browser unresponsive for 30 seconds after opening a notebook

### ISSUE THREAD

**Issue by henrik-b** [andypetrella/spark-notebook#640]
This is not a severe problem, but if you show a demo, and the browser says "The following pages have become unresponsive. You can wait for them to become responsive or kill them.", the first impression of Spark Notebook is negative. After a while, the message disappears automatically.

**Comment by vidma** [comment]
I have also noticed various slowness on larger notebooks too:
- [ ] first page load
- [ ] if cell outputs large result
- [ ] when scrolling the page

might be not easy to solve though, but it would be very nice.

**Comment by henrik-b** [comment]
Yes, the notebook becomes temporarily unresponsive during first page load and when displaying large results.

**Comment by paulcoghlan** [comment]
Hi Henrik - do you have (or can you suggest) a sample notebook you could share that demonstrates slow loading?  I'll take a look to investigate further.

**Comment by vidma** [comment]
yeah, we need a slow notebook first. 
`master` was improved by removing old sidebar.
